### ISSUE TITLE
AnalyticalTable: enter interactive content of cell via F2

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents-react#6161]
### Is your feature request related to a problem?

There should be a concept for entering interactive cell content via the default "edit mode" key (F2) similar to SAP/OpenUI5 tables.

Follow up to: https://github.com/SAP/ui5-webcomponents-react/issues/6041

### Describe the solution you'd like

Check feasibility and design for this.

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by summer2200** [comment]
Hi @Lukas742 , I'd like to reopen this issue as it's still reproducible.

**Comment by Lukas742** [comment]
Hi @summer2200 

please give more context, as for me the behavior is working as intended: https://ui5.github.io/webcomponents-react/v2/?path=/docs/data-display-analyticaltable-plugin-hooks-usef2celledit--docs

**Comment by summer2200** [comment]
When focus is on the cell which contains interactive elements, pressing F2, the focus should fall on the first interactive element inside the cell, but currently in AnalyticalTable, this cannot be achieved. I have made a recording of it, also compared with F2 behavior in Table component.

https://github.com/user-attachments/assets/f3b1f3a4-9e16-4720-ac15-76e37f397c46

**Comment by Lukas742** [comment]
@summer2200 as shown in the link I posted, you have to apply the `useF2CellEdit` hook.

**Comment by summer2200** [comment]
> Hi [@summer2200](https://github.com/summer2200)
> 
> please give more context, as for me the behavior is working as intended: https://ui5.github.io/webcomponents-react/v2/?path=/docs/data-display-analyticaltable-plugin-hooks-usef2celledit--docs

So we need to add the hook manually in our code?

**Comment by Lukas742** [comment]
Yes, as otherwise it would be a breaking change. Please see the docs, everything is explained there.

**Comment by summer2200** [comment]
Thanks a lot!

**Comment by summer2200** [comment]
Hi @Lukas742 , just need to know if Icon is also supported for F2 focusing, please see my example here:
https://stackblitz.com/edit/github-sw7uydyc?file=src%2FApp.tsx

**Comment by Lukas742** [comment]
Hi @summer2200 the `Icon` isn't currently taken into account. This was an oversight and will be fixed by this [PR](https://github.com/UI5/webcomponents-react/pull/7782). Thanks for noticing!
